requirements.yml

```diff
@@ -108,11 +108,15 @@
   numpy_compiled_extensions: false
   python: '>2.6,<3'
 
-# Chokes on no source
-# - name: gwcs
-#   version: 0.1.dev44
-#   astropy_helpers: true
-#   numpy_compiled_extensions: true
+- name: gwcs
+  version: 0.5
+  astropy_helpers: true
+  numpy_compiled_extensions: false
+
+- name: asdf
+  version: 1.0.0
+  astropy_helpers: true
+  numpy_compiled_extensions: yes
```

Comment:
yes sorry, I was just a desperate to try to make it work (and don't know much about how the package building is working).